packages/extras/src/lib/components/MeshRefractionMaterial/MeshRefractionMaterial.svelte.d.ts

```diff
@@ -0,0 +1,33 @@
+import type { Events, Props, Slots } from '@threlte/core'
+import { SvelteComponent } from 'svelte'
+import type { ColorRepresentation, CubeTexture, ShaderMaterial, Texture } from 'three'
+
+/*
+  To use this component you need to install the seperate library `three-mesh-bhv`
+	please run `npm install three-mesh-bhv` before adding this component to your project.
+*/
+
+export type MeshRefractionMaterialProps = Props<ShaderMaterial> & {
+  envMap?: CubeTexture | Texture
+  /** Number of ray-cast bounces, it can be expensive to have too many, 2 */
```

Comment:
```suggestion
  /**
   * Number of ray-cast bounces, it can be expensive to have too many
   * @default 2
   */
```
^^ JSDoc has a nice `@default` option that I think would make these prop descriptions easier to read

Reply:
👍 added